client/my-sites/plugins/plugin-details-CTA/usps.tsx

```diff
@@ -0,0 +1,114 @@
+import { Gridicon } from '@automattic/components';
+import styled from '@emotion/styled';
+import { useTranslate } from 'i18n-calypso';
+
+const StyledUl = styled.ul`
+	margin-top: 20px;
+	margin-left: 0;
+	list-style-type: none;
+`;
+
+const StyledLi = styled.li`
+	color: var( --studio-gray-50 );
+	font-size: 12px;
+	display: flex;
+	align-items: center;
+	margin: 5px 0;
+
+	.title {
+		color: var( --studio-gray-50 );
+		font-size: 14px;
+		font-weight: 500;
+		margin-top: 10px;
+	}
+
+	svg {
+		margin-right: 8px;
+	}
+`;
+
+interface Props {
+	shouldUpgrade: boolean;
+	isFreePlan: boolean;
+	isMarketplaceProduct: boolean;
+}
+
+const USPS: React.FC< Props > = ( { shouldUpgrade, isFreePlan, isMarketplaceProduct } ) => {
+	const translate = useTranslate();
+
+	const filteredUSPS = [
+		...( isMarketplaceProduct
+			? [
+					{
+						id: 'updates',
+						image: <Gridicon icon="gift" size={ 16 } />,
+						text: translate( 'Plugin updates' ),
+						eligibilities: [ 'marketplace' ],
+					},
+			  ]
+			: [] ),
+		...( isMarketplaceProduct
+			? [
+					{
+						id: 'refund',
+						image: <Gridicon icon="refund" size={ 16 } />,
+						text: translate( '30 day money-back guarantee' ),
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **30** times:
`translate( '30 day money back guarantee' )` *ES Score: 13*
See 2 additional suggestions in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/add/plugin_cta_usps)